Hue.xcodeproj/project.pbxproj

```diff
@@ -611,6 +618,8 @@
 				PRODUCT_NAME = Hue;
 				SDKROOT = macosx;
 				SKIP_INSTALL = YES;
+				SWIFT_OPTIMIZATION_LEVEL = "-Owholemodule";
```

Comment:
I was just curious, but now that you mention it I think they talked about that in the Xcode debugging session this WWDC 😎